.github/workflows/PR-CI.yml

```diff
@@ -89,8 +89,11 @@ jobs:
           HASURA_GRAPHQL_ADMIN_SECRET: so-secretish
           HASURA_PORT: 8080
 
+      - name: Debug docker containers
+        run: docker ps -a
+
       - name: Build Web
-        run: yarn web:build
+        run: yarn wait-on http://localhost:4000/healthz && yarn web:build
```

Comment:
Do you think we should wait for backend at port 4000 or hasura at port 8080?

Reply:
Good question. Not sure if hasura has a similar built-in health check. This is already better than it was, so I think if we are still seeing issues we can point it to hasura insteadl